kubejs/server_scripts/recipes.js

```diff
@@ -0,0 +1,3 @@
+ServerEvents.recipes(event => {
+    event.remove({ output: 'strainers:summoning_block' })
```

Comment:
alternatively we could install https://www.curseforge.com/minecraft/mc-mods/item-obliterator

Reply:
i personally like the idea of handling it through kubejs instead of installing more mods